content/nginx-one/about.md

```diff
@@ -1,7 +1,7 @@
 ---
 description: ''
 docs: DOCS-1392
-title: About
+title: Manage your NGINX fleet
```

Comment:
I wonder if we can tuck the legal blurb at the bottom of this doc somewhere else -- it's a bit awkward to lead with it.

Right now it reads like:
“Here are the great things you can do with NGINX One. But first, a word from the lawyers.”

Reply:
Created https://github.com/nginx/documentation/issues/625